package.json

```diff
@@ -8,10 +8,37 @@
     "node": ">=6.13.0"
   },
   "scripts": {
```

Comment:
I recommend alphabetizing key-value pairs of an object unless you have a compelling reason to do otherwise.